cdist/config.py

```diff
@@ -109,6 +112,19 @@ def hosts(source):
                     yield host
 
     @classmethod
+    def _check_beta(cls, args_dict):
```

Comment:
Shoudn't this one be in scripts/cdist?

Especially if we want to have the same functionality in inventory.

We could also create a new exception CdistBetaRequired and throw it in the commandline parser of the sub commands and handle this in scripts/cdist